staging/src/k8s.io/apimachinery/pkg/util/diff/cmp.go

```diff
@@ -0,0 +1,30 @@
+//go:build usegocmp
+// +build usegocmp
```

Comment:
Is this something that can be enabled also when building Kubernetes?

Reply:
yes. using `GOFLAGS`.

Reply:
you will see the binary size ballon up when you do though.